Fix duplicated screenshots in the Fine grain admin permissions sectio… (#33314)
keycloak/keycloak: docs/documentation/server_admin/topics/admin-console-permissions/fine-grain.adoc

When first initialized the `manage` permission does not have any policies associated with it.
You will need to create one by going to the policy tab.  To get there fast, click on
the `Client details` link shown in the above image. Then click on the policies tab.

> There's a button on this page called `Create client policy`.  There's a multitude of policies
> you can define.  You can define a policy that is associated with a role or a group or even define
> rules in JavaScript.  For this simple example, we're going to create a `User Policy`.

Reviewer: ```suggestion
On this page, look for the `Create client policy` button, which you can use to define many policies.  You can define a policy that is associated with a role or a group or even define
rules in JavaScript.  For this simple example, we are going to create a `User Policy`.
```

.Assign user policy
image:images/fine-grain-client-assign-user-policy.png[Fine grain client assign user policy]

The `sales-admin` user now has permission to manage the `sales-application` client.

> There's one more thing we have to do.  Go to `Users`, select the `sales-admin` user, then go to the `Role Mappings` tab and assign the `query-clients` role to the user.

Reviewer: ```suggestion
There is one more thing we have to do.  Go to `Users`, select the `sales-admin` user, then go to the `Role Mappings` tab and assign the `query-clients` role to the user.
```
